static/engine/Sprite.js

```diff
@@ -25,29 +26,25 @@ export class Sprite {
 
 	}
 
+
 	draw(ctx) {
```

Comment:
don't edit this method. why did you edit it ?

Reply:
o ye nvm, but why does the git diff show that the entire method is different?

(why) did you remove "if (this.spriteSheet.complete) {"?